app/src/main/java/uk/gov/onelogin/network/auth/AuthCodeExchangeModule.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.onelogin.network.auth
+
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+interface AuthCodeExchangeModule {
```

Comment:
This interface definition is used by Hilt to expose `AuthCodeExchange` as an App-wide singleton class